### DISCUSSION TITLE
Simple Value from MySQL query

### DISCUSSION THREAD

**Discussion by doucwiki** [Budibase/budibase#9545] [Help]
Hi,

got a problem.
I have a connected MySQL instance and need to display a simple MySQL query as a value in Budibase
Table is 'Patients' with admission date and release date, Query is

SELECT DATEDIFF(patients.`release`, patients .`admission`)as {{GFK_duration}} FROM patients where {{GFK_foreignkey}};

bindings are GFK_duration --> days and GFK_foreignkey --> 1

Running the Query in Budibase gives me the correct results:

{
  "days": "5"
}
etc. continuing for the whole patients table

I need to add a component in Budibase like a card which is showing  the Patient (for example NAME) and the corresponding days value calculated from the above query.
I tried nested data providers (patients table and the query) but didn't get it working.

Since I am no IT pro I ran into a wall.
Help would be appreciated!!

**Comment by melohagan** [comment]
Hey @doucwiki 

Could you not return the patient columns you want to display in your custom query? So that you are returning Patient Name and the Days from your SQL.
